src/Optimize/VariableSet.cpp

```diff
@@ -271,7 +273,8 @@ void VariableSet::print(std::ostream& os, int leftPadSpaces, bool printHeader)
   for (int i = 0; i < NameAndValue.size(); ++i)
   {
     os << pad_str << setw(max_name_len) << NameAndValue[i].first << " " << std::setprecision(6) << std::scientific
-       << setw(max_value_len) << NameAndValue[i].second << " " << setw(max_type_len) << ParameterType[i].second << " "
+       << setw(max_value_len) << std::real(NameAndValue[i].second) << " " << setw(max_value_len) << std::imag(NameAndValue[i].second) << " "
```

Comment:
If we build the code real, we still see the column of Value Imag.
The old way should print complex number as `(0.1233,0.12134)`.
I think the old way works better.

Reply:
Hi Ye, doing it in the old way would require different codes for real and complex optimize unit test in test_variable_set.cpp since their outputs are different. Is this something that we want to avoid?

Reply:
You can ifdef the reference output string content.